src/lib.rs

```diff
@@ -37,18 +37,18 @@
 //! # Dependencies
 //!
 //! The application crate must depend on a device crate generated using
-//! [`svd2rust`] v0.11.x and the "rt" feature of that crate must be enabled. The
+//! [`svd2rust`] v0.12.x and the "rt" feature of that crate must be enabled. The
 //! SVD file used to generate the device crate *must* contain [`<cpu>`]
 //! information.
 //!
-//! [`svd2rust`]: https://docs.rs/svd2rust/0..0/svd2rust/
+//! [`svd2rust`]: https://docs.rs/svd2rust/0.12.0/svd2rust/
 //! [`<cpu>`]: https://www.keil.com/pack/doc/CMSIS/SVD/html/elem_cpu.html
 //!
 //! # `app!`
 //!
 //! The `app!` macro is documented [here].
 //!
-//! [here]: https://docs.rs/cortex-m-rtfm-macros/0.2.0/cortex_m_rtfm_macros/fn.app.html
+//! [here]: https://docs.rs/cortex-m-rtfm-macros/0.2.1/cortex_m_rtfm_macros/fn.app.html
```

Comment:
You bumped the version to `0.3.0` but this refers to `0.2.1`

Reply:
:-(